tox.ini

```diff
@@ -38,3 +38,9 @@ usedevelop = True
 commands =
     python -m pip list --format=columns
     python -c 'print("\n\nTo activate type: `source venv/bin/activate`\n\n")'
+
+[pylint]
+max-line-length = 120
+
+[flake8]
+max-line-length = 120
```

Comment:
This one is set here:

https://github.com/r-bioinformatics/edgePy/blob/968b163d1e163be54d4ec8d94a5abc31c69d49e5/setup.cfg#L48

Can you configure `pylint` in `setup.cfg` too? I'm not sure.

Also have a description of the style here which will need updating or removal if we rather an implicit rule (as checked by tests only).

Reply:
It works for me - and solves a "line too long error" thrown by pylint in tox.

Reply:
Right, but we're trying to define environments in `tox.ini` and tool configuration in `setup.cfg`.

Reply:
Yes - wasn't saying I won't make the change in setup.cfg, just confirming that it also works.

Reply:
Ok, can confirm that the lines in tox.ini aren't needed.  Removed.